package.json

```diff
@@ -24,6 +24,7 @@
     "@newfold-labs/js-utility-ui-analytics": "^1.2.0",
     "@newfold-labs/wp-module-runtime": "^1.0.0",
     "@newfold/ui-component-library": "^1.0.0",
+    "@newfold/wp-module-facebook": "file:../wp-module-facebook",
```

Comment:
Looks like this will need to be updated before release

Reply:
yes, we'll update it before the release.